### DISCUSSION TITLE
Option to hide using develop branch warning.

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#10321] [Ideas & Feature requests]
Would be nice to have an option in "/General Configuration/Advanced Settings/GitHub" to hide develop branch notification and instead add a small branch icon in top of the page some where next to Medusa icon.
So those like me whom mostly are on Dev branch don't have to see that warning all the time.

**Comment by p0psicles** [comment]
If you open your config.ini file. There is a setting named developer. Set it to 1. Then restart.
It should not show the branch warning.
